__mocks__/readme.md

```diff
@@ -0,0 +1,10 @@
+## How to get started
+
+### Install Camouflage by running:
+- npm install -g camouflage-server
```

Comment:
could you please put command line into a [fenced code blocks](https://www.markdownguide.org/extended-syntax/#fenced-code-blocks)?

Reply:
Done